src/components/seekbar-live-playback-container/seekbar-live-playback-container.js

```diff
@@ -78,7 +78,7 @@ class SeekBarLivePlaybackContainer extends Component {
         updateCurrentTime={data => this.props.updateCurrentTime(data)}
         isDvr={this.props.isDvr}
         currentTime={this.props.currentTime}
-        duration={this.props.duration}
+        duration={this.props.player.isOnLiveEdge() ? this.props.currentTime : Math.max(this.props.duration, this.props.currentTime)}
```

Comment:
```suggestion
        duration={this.props.player.isOnLiveEdge() ? this.props.currentTime : this.props.duration}
```
`currentTime > duration` means `isOnLiveEdge()` is `true` so the `Math.max` is unnecessary

Reply:
No, because the check of the isOnLiveEdge is done on seeking. But it can be that in seeking the currentTime was smaller than duration (hence isOnLiveEdge = false) but because segments are being downloaded and duration "jumps" than its possible currentTime will be greater than duration and it will exceed 100%. This is another fail safe mechanism